pkg/proxy/winkernel/proxier.go

```diff
@@ -144,41 +142,16 @@ type remoteSubnetInfo struct {
 }
 
 const NETWORK_TYPE_OVERLAY = "overlay"
+const NETWORK_TYPE_L2BRIDGE = "L2Bridge"
 
-func newHostNetworkService() (HostNetworkService, hcn.SupportedFeatures) {
-	var hns HostNetworkService
-	hns = hnsV1{}
+func newHostNetworkService() (IHCNUtils, hcn.SupportedFeatures) {
+	var h IHCNUtils
 	supportedFeatures := hcn.GetSupportedFeatures()
 	if supportedFeatures.Api.V2 {
-		hns = hnsV2{}
+		h = hcnutils{&RealHCN{}}
```

Comment:
What's the failure mode here? Looks like we've dropped support for hnsv1?  If that is the case that should probably come in a separate pr?  What happens if `supportedFeatures.Api.V2` isn't supported?